ansible_bender/cli.py

```diff
@@ -310,6 +311,24 @@ def _build(self):
     def _build_inside_openshift(self):
         build_inside_openshift(self.app)
 
+    def fancy_time(build_time):
```

Comment:
can we move the function to utils so that it could be used across the project?

Reply:
Done 😄